.npmignore

```diff
@@ -1,2 +0,0 @@
-node_modules
-.idea
```

Comment:
@florina-chesaru Can you please also re-add these files.

Reply:
Done, sry for that.